source/scss/1977.scss

```diff
@@ -10,7 +10,7 @@
   filter: contrast(1.1) brightness(1.1) saturate(1.3);
 
   &:after{
-    background: rgba(243, 106, 188, .3);
+    background: rgba(243, 106, 188, 0.3);
```

Comment:
Oh. You did not do it everywhere, so I just assumed it was better to add it.

My 2 cents on this is that it is the minifier’s role to shorten your code, not yours. :)